// ng2-pdf-viewer/esm5/src/app/pdf-viewer/pdf-viewer.module.js
import { __decorate } from "tslib";
/**
 * Created by vadimdez on 01/11/2016.
 */
import { NgModule } from '@angular/core';
import { PdfViewerComponent } from './pdf-viewer.component';
import * as ɵngcc0 from '@angular/core';
var PdfViewerModule = /** @class */ (function () {
    function PdfViewerModule() {
    }
PdfViewerModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: PdfViewerModule });
PdfViewerModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function PdfViewerModule_Factory(t) { return new (t || PdfViewerModule)(); } });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(PdfViewerModule, { declarations: function () { return [PdfViewerComponent]; }, exports: function () { return [PdfViewerComponent]; } }); })();
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(PdfViewerModule, [{
        type: NgModule,
        args: [{
                declarations: [PdfViewerComponent],
                exports: [PdfViewerComponent]
            }]
    }], function () { return []; }, null); })();
    return PdfViewerModule;
}());
export { PdfViewerModule };
